docs/how-to/examples.md

```diff
@@ -134,12 +134,8 @@ dataset.date_of_death = patients.date_of_death
 dataset.define_population(patients.exists_for_patient())
 ```
 
-:notepad_spiral: This value comes from the patient's EHR record.
+:notepad_spiral: This value comes from the patient's EHR record. You can find more information about the accuracy of this value in the [reference schema](https://docs.opensafely.org/ehrql/reference/schemas/beta.core/#recording-of-death-in-primary-care).
```

Comment:
While the absolute path that @iaindillingham suggested works, I'd actually use the relative path to the file:

```suggestion
:notepad_spiral: This value comes from the patient's EHR record. You can find more information about the accuracy of this value in the [reference schema](../reference/schemas/beta.core.md#recording-of-death-in-primary-care).
```

This points explicitly to the file which:

* shows clearly that the resource being linked to is another documentation source file
* helps when checking the URLs are valid with a link checker[^1]
* avoids adding an extra MkDocs log line like:
  ```
  INFO    -  Doc file 'how-to/examples.md' contains an absolute link '/reference/schemas/beta.core/#recording-of-death-in-primary-care', it was left as is. Did you mean
           '../reference/schemas/beta.core.md#recording-of-death-in-primary-care'?
  ```

[^1]: The link checker only checks source files. It won't find the resource in the link. But it can if it knows that it's looking for a relative file path.